lib/net/imap.rb

```diff
@@ -906,6 +928,10 @@ def uid_store(set, attr, flags)
     # of the specified destination +mailbox+. The +set+ parameter is
     # a number, an array of numbers, or a Range object. The number is
     # a message sequence number.
+    #
+    # If the server supports the UIDPLUS extension it returns an array with
```

Comment:
Adding a reference link.  🙂

```suggestion
    # If the server supports the [UIDPLUS[https://www.rfc-editor.org/rfc/rfc4315.html]]
    # extension, it returns an array with
```

n.b. this messes up the paragraph's line-breaks, but GitHub still doesn't allow multi-line suggestions.  😉